lib/core/src/wallet.rs

```diff
@@ -214,7 +212,7 @@ impl OnchainWallet for LiquidOnchainWallet {
                 })?,
                 amount_sat,
             )?
-            .fee_rate(fee_rate_sats_per_kvb)
```

Comment:
IIUC removing this will result in using the default fee rate of 0.1 sat/vbyte, which is what we want. But would it make sense to still define it from `LIQUID_FEE_RATE_SAT_PER_VBYTE`? That way, we are sure no issues will arise in the future from some change in the LWK default fee rate.

Reply:
Makes sense [51d7d63](https://github.com/breez/breez-sdk-liquid/pull/637/commits/51d7d63a51ffd157c0a56372cfcff0cd9f5f95c5)